content/en/news/releases/1.12.x/announcing-1.12/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Announcing Istio 1.12
+linktitle: 1.12
+subtitle: Major Update
+description: Istio 1.12 release announcement.
+publishdate: 2021-11-16
+release: 1.12.0
+skip_list: true
+aliases:
+    - /news/announcing-1.12
+    - /news/announcing-1.12.0
+---
+
+We are pleased to announce the release of Istio 1.12!
+
+{{< relnote >}}
+
+This is the last release of 2021. We would like to thank the entire Istio community, and especially the release managers [Daniel Grimm](https://github.com/dgn) from Red Hat and [Kenan O'Neal](https://github.com/Kmoneal) from Aspen Mesh, for helping to get 1.12.0 published.
+
+{{< tip >}}
+Istio 1.12.0 is officially supported on Kubernetes versions `1.18` to `1.22`.
```

Comment:
Note my earlier comment we also need to update the supported versions in data/args.yaml and istio.io/istio.io/content/en/docs/releases/supported-releases/index.md.